spec/models/book_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+describe Book do
+  let(:book) { build(:book) }
+
+  context 'When creates a book' do
+    it 'validates presence of :gender' do
+      is_expected.to validate_presence_of(:gender)
+    end
+
+    it 'validates presence of :gender' do
+      is_expected.to validate_presence_of(:gender)
+    end
+
+    it 'validates presence of :author' do
+      is_expected.to validate_presence_of(:author)
+    end
+
+    it 'validates presence of :image' do
+      is_expected.to validate_presence_of(:image)
+    end
+
+    it 'validates presence of :title' do
+      is_expected.to validate_presence_of(:title)
+    end
+
+    it 'validates presence of :editor' do
+      is_expected.to validate_presence_of(:editor)
+    end
+
+    it 'validates presence of :year' do
+      is_expected.to validate_presence_of(:year)
+    end
+  end
```

Comment:
we always check if the entity we build is valid as is, using something like:
`it { is_expected.to be_valid }`
That way you make sure that whatever you are building with the factory makes sense with the model